packages/web/src/tracking/tracking.ts

```diff
@@ -0,0 +1,11 @@
+import { OccurrenceDetails } from '@openfeature/core';
```

Comment:
```suggestion
import types { OccurrenceDetails } from '@openfeature/core';
```

Reply:
`import type`

Reply:
I've fixed this is a separate PR with a new eslint rule, nice catch.